app/views/devise/registrations/new.html.haml

```diff
@@ -0,0 +1,37 @@
+%header.entry-header
+  %h1.entry-title Ansök om medlemskap
+.post-title-divider
+  %span
+.entry-content.wpcf7
+  = form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
+    = devise_error_messages!
+    - f.label :first_name, 'First Name'
+    = f.text_field :first_name, placeholder: 'Förnamn', class: 'wpcf7-form-control wpcf7-text wpcf7-validates-as-required'
+    - f.label :last_name
+    = f.text_field :last_name, placeholder: 'Efternamn', class: 'wpcf7-form-control wpcf7-text wpcf7-validates-as-required'
+    - f.label :street
+    = f.text_field :street, placeholder: 'Gatuadress (hela)', class: 'wpcf7-form-control wpcf7-text wpcf7-validates-as-required'
+    - f.label :postal_code
+    = f.text_field :postal_code, placeholder: 'Postnummer', class: 'wpcf7-form-control wpcf7-text wpcf7-validates-as-required'
+    - f.label :city
+    = f.text_field :city, placeholder: 'Ort', class: 'wpcf7-form-control wpcf7-text wpcf7-validates-as-required'
+    - f.label :phone
+    = f.telephone_field :phone, placeholder: 'Telefon (0701223344)', class: 'wpcf7-form-control wpcf7-text wpcf7-validates-as-required'
+    %br/
+    - f.label :email
+    = f.email_field :email, placeholder: 'din@email.se', class: 'wpcf7-form-control wpcf7-text wpcf7-validates-as-required'
+    - f.label :email_confirmation
```

Comment:
I was not able to get this to work on my system.  (No 'email_confirmation' method found for User).  Is there  code missing that compares the `email` and `email_confirmation` fields? (like in the controller)?

Ditto with `password` and `password_confirmation` -- seems that there is some implementation  missing that (a) defines this somewhere (as a method, etc), and (b) makes use of it as part of the validation when the form is submitted.

Is it really _necessary_ for MVP1 ?